### DISCUSSION TITLE
Is it possible to apply regulation to basic auth?

### DISCUSSION THREAD

**Discussion by christaikobo** [authelia/authelia#7989] [Q&A] [answered]
When using basic auth, it seems that the regulation "ban user" is not working. Is it possible to apply regulation to basic auth?

Thanks!

**Comment by james-d-elliott** [comment, marked as answer]
It actually isn't supported in the current version, but I also incidentally noticed this in the regulation rework I've been doing which will land in 4.39, and decided it was a good addition. I have a functional implementation however I've not added tests so it's yet to be rebased and pushed.
